lib/happo/public/HappoDiffs.jsx

```diff
@@ -81,20 +81,66 @@ function HappoNewImages({ images }) {
   );
 }
 
-function HappoDiff({ image }) {
-  return (
-    <div>
-      <HappoImageHeading
-        image={image}
-      />
-      <div className='happo-diff__images'>
-        <img src={image.previous} />
+const HappoDiff = React.createClass({
```

Comment:
It would be a little nicer to use classes here instead of the old createClass syntax.